guides/micronaut-java-records/java/src/test/resources/application-test.yml

```diff
@@ -0,0 +1,4 @@
+datasources:
+    default:
+        url: jdbc:tc:postgresql:12:///postgres
```

Comment:
Maybe it's worth mentioning that this is a special url that Testcontainers knows how to manage to start automatically a postgres container for you.